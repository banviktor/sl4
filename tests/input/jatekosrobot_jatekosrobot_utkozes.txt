randomization off
questions off
start_game 5
2 4
3 1
5 8
7 8
2 9
jump
inputvector 0 1
jump
inputvector 1 0
jump
inputvector -1 0
jump
list_player_robots
jump
N
inputvector 1 0
jump
inputvector 0 1
jump
jump
N
list_player_robots
jump
exit
