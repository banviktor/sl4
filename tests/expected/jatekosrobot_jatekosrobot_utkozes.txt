Véletlenszerû események kikapcsolva.
Kérdezés kikapcsolva.
Pálya betöltve.
Új játék elkezdve 5 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Új pozíció: 2.0, 4.0
Ez a(z) 1. kör.
A Sárga Robot következik.
Inputvector: 0.0, 1.0
Új pozíció: 3.0, 2.0
Ez a(z) 1. kör.
A Zöld Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 6.0, 8.0
Ez a(z) 1. kör.
A Kék Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 6.0, 8.0
Ez a robot a következõkkel ütközött:
    Zöld
Ez a(z) 1. kör.
A Lila Robot következik.
Játékosrobotok listája:
	Piros: 2.0, 4.0
	Sárga: 3.0, 2.0
	Lila: 2.0, 9.0
Új pozíció: 2.0, 9.0
Ez a(z) 2. kör.
A Piros Robot következik.
Inputvector: 1.0, 0.0
Új pozíció: 3.0, 4.0
Ez a(z) 2. kör.
A Sárga Robot következik.
Inputvector: 0.0, 1.0
Új pozíció: 3.0, 4.0
Ez a robot a következõkkel ütközött:
    Piros
Ez a(z) 2. kör.
A Lila Robot következik.
Új pozíció: 2.0, 9.0
Ez a(z) 3. kör.
A Sárga Robot következik.
Játékosrobotok listája:
	Sárga: 3.0, 4.0
	Lila: 2.0, 9.0
Új pozíció: 3.5, 5.0
Ez a(z) 3. kör.
A Lila Robot következik.
